.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python: 
+  - 3.6
+install:
+  - pip install -r backend/requirements.txt
+script:
+  - coverage run -m pytest backend/

```

Comment:
can we run `pytest --cov=scrumsistant backend/tests` instead?  (or will it respect the pytest.ini?)